src/lib/journey/callbacks/hidden-value/hidden-value.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+  import type { HiddenValueCallback } from '@forgerock/javascript-sdk';
+
+  export let callback: HiddenValueCallback;
+  export let idx: number;
+
+  const inputName = callback?.payload?.input?.[0].name || `hidden-value-${idx}`;
+  const inputValue = callback?.getInputValue();
+</script>
+
+<input type="hidden" name={inputName} id={inputName} value={inputValue} />
```

Comment:
As you can see we're not rendering much here - but I think it's important that we do render something, so that scripts could be supported in future.

Reply:
Understood. Honestly, I'm struggling with supporting this "legacy" style of handling customization. I touched on this above, but I'll reiterate it here as I think this is really important for us to seriously consider.

If you think about the _core_ reason for this pattern – sending scripts from AM for a client to execute in the DOM and injecting the side-effect of this execution into a hidden input – it's all about customizing the client's behavior when you _don't have access to the source code_. 

This was always true for XUI and is still somewhat true for Platform Login. Customers don't have a holistically designed process for customizing or adding features to a login flow that is not natively supported by the client application, and they can't easily alter said application. 

The Login Widget is directly designed to address this. We are explicitly designing a framework that allows them to add, extend, customize the login flow directly in the client application. Through good design, education and evangelism, I feel we can effectively "flip" the mental model of how our customers control their UX on its head and provide a far superior developer experience while doing so.

Anyway, I'm happy to continue this conversation with you and the rest of the team. I think it's vital for us to deeply understand this concept and evaluate this feature without the historical bias of its legacy. If, after deep contemplation, we still have a strong reason to allow for this "legacy" (as I call it) feature, then I'm happy to support it :)

Reply:
I understand your point of view - and I agree, when it comes to allowing scripting for the sake of modifying the UI. However my concern is related to the partner integrations - the nodes of some of these return a client side script to render something and may expect a hidden value callback to render something on the page to capture the result. I think for this change, perhaps we can decide to render nothing, however I think it merits further discussion.

Reply:
Absolutely agree with discussing this further as I'd love to know more about these partner integrations you mention.